src/main/webapp/ts/profileBuilder.ts

```diff
@@ -1,39 +1,71 @@
-type authInfoFromServlet = {
+type authInfo = {
     loginUrl: string;
     logoutUrl: string;
     hasProfile: boolean;
 };
 
 function setForm(value: string) {
-    const menteeForm =  document.getElementById('Mentee');
+    const menteeForm = document.getElementById('Mentee');
     const mentorForm = document.getElementById('Mentor');
     const userTypeButton = document.getElementById('user-input');
 
-    if(menteeForm && mentorForm && userTypeButton){
-        if(value == 'Mentee') {
-            menteeForm.setAttribute('style','display:block;');
-            mentorForm.setAttribute('style','display:none;');
-            userTypeButton.setAttribute('value','Mentee');
+    const menteeInputs:string[] = ['name-input','school-input', 'major-input' ];
+    const mentorInputs:string[] =['profName-input','company-input','careerTitle-input', 'bio-input', 'specialty-input']
+
+    if (menteeForm && mentorForm && userTypeButton) {
+        if (value == 'Mentee') {
+            menteeForm.setAttribute('style', 'display:block;');
+            mentorForm.setAttribute('style', 'display:none;');
+            userTypeButton.setAttribute('value', 'Mentee');
+
+            if (menteeInputs && mentorInputs) {
```

Comment:
What is this if statement checking?

Reply:
It's checking if the array is full or null. I should be able to remove it.

Reply:
I removed it.